contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -848,74 +848,54 @@ get_authid_login_ext_idx_oid(void)
  *			USER EXT
  *****************************************/
 
-bool
-is_user(Oid role_oid)
+/*
+ * Check if role is a bbf db principal. Returns BBF_ROLE if it is
+ * a db role, returns BBF_USER if it is db user else returns 0
+ * Looks only in current bbf db when current_db_only is set to true
+ */
+
+const int
+is_database_principal(Oid role_oid, bool current_db_only)
```

Comment:
Right, changed name to `get_db_principal_kind`. Did not want to use "role" in function name since it can be used for both pltsql database users and roles.
Also changes the bool flag to const char *db_name.